app/auth/mutations/resetPassword.test.ts

```diff
@@ -1,20 +1,22 @@
 import resetPassword from "./resetPassword"
 import db from "db"
 import { hash256, SecurePassword } from "blitz"
+import * as login from "./login"
 
 beforeEach(async () => {
   await db.$reset()
 })
 
 const mockCtx: any = {
   session: {
-    $create: jest.fn,
+    $create: jest.fn(),
```

Comment:
why the brackets?

Reply:
As i just noticed, `jest.fn()` must be called. Otherwise we cannot access the values.